stdlib/public/core/FloatingPoint.swift.gyb

```diff
@@ -1011,94 +1011,85 @@ public protocol FloatingPoint: SignedNumeric, Strideable, Hashable {
   /// - If `x` is `-greatestFiniteMagnitude`, then `x.nextDown` is `-infinity`.
   var nextDown: Self { get }
 
-  /// Returns a Boolean value indicating whether this instance is equal to the
-  /// given value.
+  /// Returns a Boolean value indicating whether `lhs` is equal to `rhs`.
   ///
-  /// This method serves as the basis for the equal-to operator (`==`) for
-  /// floating-point values. When comparing two values with this method, `-0`
-  /// is equal to `+0`. NaN is not equal to any value, including itself. For
-  /// example:
+  /// When comparing two values with this method, `-0` is equal to `+0`. NaN
+  /// is not equal to any value, including itself. For example:
   ///
   ///     let x = 15.0
-  ///     x.isEqual(to: 15.0)
+  ///     x == 15.0
   ///     // true
-  ///     x.isEqual(to: .nan)
+  ///     x == .nan
   ///     // false
-  ///     Double.nan.isEqual(to: .nan)
+  ///     Double.nan == .nan
   ///     // false
   ///
-  /// The `isEqual(to:)` method implements the equality predicate defined by
+  /// The `==` operator implements the equality predicate defined by
   /// the [IEEE 754 specification][spec].
   ///
   /// [spec]: http://ieeexplore.ieee.org/servlet/opac?punumber=4610933
   ///
-  /// - Parameter other: The value to compare with this value.
-  /// - Returns: `true` if `other` has the same value as this instance;
-  ///   otherwise, `false`. If either this value or `other` is NaN, the result
-  ///   of this method is `false`.
-  func isEqual(to other: Self) -> Bool
+  /// - lhs: The first value to compare.
+  /// - rhs: The second value to compare.
+  /// - Returns: `true` if `lhs` and `rhs` represent the same extended real
+  ///   number value; otherwise, `false`. If either `lhs` or `rhs` is NaN,
+  ///   the result of this oeprator is `false`.
+  static func ==(_ lhs: Self, _ rhs: Self) -> Bool
 
-  /// Returns a Boolean value indicating whether this instance is less than the
-  /// given value.
-  ///
-  /// This method serves as the basis for the less-than operator (`<`) for
-  /// floating-point values. Some special cases apply:
+  /// Returns a Boolean value indicating whether `lhs` is less than `rhs`.
+  /// Some special cases apply:
   ///
-  /// - Because NaN compares not less than nor greater than any value, this
-  ///   method returns `false` when called on NaN or when NaN is passed as
-  ///   `other`.
-  /// - `-infinity` compares less than all values except for itself and NaN.
-  /// - Every value except for NaN and `+infinity` compares less than
-  ///   `+infinity`.
+  /// - Because NaN is incomparable with any value, this method returns `false`
```

Comment:
Instead of ``this method returns `false` when`` let's say ``the result of this comparison is `false` when``, throughout.